ast/src/statements/console/formatted_string.rs

```diff
@@ -18,14 +18,43 @@ use crate::{Expression, Node, Span};
 
 use serde::{Deserialize, Serialize};
 use std::fmt;
-use tendril::StrTendril;
 
 #[derive(Clone, PartialEq, Eq, Serialize, Deserialize, Debug)]
 pub enum FormatStringPart {
-    Const(#[serde(with = "crate::common::tendril_json")] StrTendril),
+    Const(char),
```

Comment:
can we still aggregate `const + const = const` here? seems like this is unnecessary and cumbersome (to have each char its own entry)